app/views/tag_set_nominations/_tag_nominations_by_fandom.html.erb

```diff
@@ -1,14 +1,14 @@
 <% @tag_set_nomination.fandom_nominations.each_with_index do |nom, index| %>
   <%= f.fields_for :fandom_nominations, nom do |nom_form| %>
     <fieldset class="tagset">
-      <legend><%= ts("Fandom %{index}", :index => index+1) %></legend>
+      <legend><%= ts("Fandom %{index}", index: index + 1) %></legend>
```

Comment:
Question! Why did this need to be changed but not `:index => index+1` on line 6?

Reply:
Oh, it doesn't! I just changed the syntax here because I looked at this line to double-check that I had the right index number for line 11, and noticed the hash-rocket syntax (and spacing). I never noticed that line 6 had a similar issue.

I can revert this line or fix line 6, whichever you prefer.

Reply:
If you could fix line 6, too, that'd be great!